warehouse/models/mart/gtfs_quality/fct_rt_feed_fetch_errors.sql

```diff
@@ -0,0 +1,24 @@
+{{ config(materialized='table') }}
+
+WITH fct_rt_feed_fetch_errors AS (
```

Comment:
I think it might be useful to extract the URL from the tags and cluster on that; also I'm now realizing we don't actually tag with the `tick` which will make joins harder than they should be 🤔 unless we just care about top-level counts anyways